docs/flink-ddl.md

```diff
@@ -165,7 +165,7 @@ Currently, it does not support computed column and watermark definition etc.
 #### `PRIMARY KEY`
 
 Primary key constraint can be declared for a column or a set of columns, which must be unique and do not contain null.
-It's required for [`UPSERT` mode](../flink/flink-writes.md#upsert).
+It's required for [`UPSERT` mode](../flink-writes.md#upsert).
```

Comment:
Do we need the `..`?

Reply:
Not sure but I'm following other links

Reply:
just FYI, you can verify if the links actually work on the deployed website by running: https://github.com/apache/iceberg-docs#running-locally